frontend/src/middlewares/error-handling/error-handling-middleware.ts

```diff
@@ -0,0 +1,22 @@
+import { type Middleware, type PayloadAction } from '@reduxjs/toolkit';
```

Comment:
Move file to frontend/src/libs/packages/store/middlewares/handle-error.middleware.ts